src/Fmt_ast.ml

```diff
@@ -2736,6 +2788,7 @@ and fmt_module_type c ({ast= mty} as xmty) =
   let ctx = Mty mty in
   let {pmty_desc; pmty_loc; pmty_attributes} = mty in
   let c = update_config c pmty_attributes in
+  (*maybe_disabled c pmty_loc @@ fun () ->*)
```

Comment:
?

Reply:
This should return a `block`. I did have time to look at this.